tests/System.Buffers.Primitives.Tests/MemoryTests.cs

```diff
@@ -167,59 +167,6 @@ private static IEnumerable<object[]> MakePermutations(params object[][] argss)
         }
         #endregion
 
-        public static IEnumerable<object[]> ReservationPerformanceData =>
```

Comment:
Initially I wanted to, but they I started to wonder if the tests are that useful given the API is now op-in. But I can add this test back, if you think it's useful.